### PR TITLE
Fix passing $length=null to grapheme_substr()

### PR DESCRIPTION
**Pull request by nicolas-grekas**

### CODE REVIEW

File: tests/Intl/Grapheme/GraphemeTest.php [hunk 1]
```diff
@@ -89,6 +89,10 @@ public function testGraphemeSubstr()
     {
         $c = 'déjà';
 
+        if (\PHP_VERSION_ID >= 70000) {
```
Comment (stof) [inline tests/Intl/Grapheme/GraphemeTest.php:92]: why only on PHP 7.0+ when the polyfill does not have such condition ?

File: src/Intl/Grapheme/Grapheme.php [hunk 2]
```diff
@@ -105,8 +105,12 @@ public static function grapheme_strlen($s)
         return 0 === $len && '' !== $s ? null : $len;
     }
 
-    public static function grapheme_substr($s, $start, $len = 2147483647)
+    public static function grapheme_substr($s, $start, $len = null)
```
Comment (stof) [inline src/Intl/Grapheme/Grapheme.php:108]: do we even need an optional argument here ? the polyfill function always provides it.

Comment (nicolas-grekas) [reply src/Intl/Grapheme/Grapheme.php:108]: actually, this is required for testing.

File: tests/Intl/Grapheme/GraphemeTest.php (hunk 1 above)
Comment (nicolas-grekas) [reply tests/Intl/Grapheme/GraphemeTest.php:92]: reverted: the behavior depends also on the version of the intl extensions. The implementation is now unconditional (but tests are for practical reasons)

**Pull request merged**